gcloud-java-pubsub/baseline/src/main/java/com/google/gcloud/pubsub/spi/v1/SubscriberSettings.java

```diff
@@ -336,4 +201,212 @@ public Object extractNextToken(ListSubscriptionsResponse payload) {
               return payload.getSubscriptionsList();
             }
           };
+
+  public static class Builder extends ServiceApiSettings.Builder {
+    private final ImmutableList<ApiCallSettings.Builder> methodSettingsBuilders;
+
+    private SimpleCallSettings.Builder<Subscription, Subscription> createSubscriptionSettings;
+    private SimpleCallSettings.Builder<GetSubscriptionRequest, Subscription>
+        getSubscriptionSettings;
+    private PageStreamingCallSettings.Builder<
+            ListSubscriptionsRequest, ListSubscriptionsResponse, Subscription>
+        listSubscriptionsSettings;
+    private SimpleCallSettings.Builder<DeleteSubscriptionRequest, Empty> deleteSubscriptionSettings;
+    private SimpleCallSettings.Builder<ModifyAckDeadlineRequest, Empty> modifyAckDeadlineSettings;
+    private SimpleCallSettings.Builder<AcknowledgeRequest, Empty> acknowledgeSettings;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Private field 'pullSettings' could be made final; it is only initialized in the declaration or constructor.](https://www.codacy.com/app/mziccard/gcloud-java/file/2381705098/issues/source?bid=3123109&fileBranchId=3225976#l217)